CCDB/test/testCcdbApi.cxx

```diff
@@ -421,16 +421,24 @@ BOOST_AUTO_TEST_CASE(TestHeaderParsing)
 
 BOOST_AUTO_TEST_CASE(TestFetchingHeaders, *utf::precondition(if_reachable()))
 {
+  // first store the object
+  string objectPath = basePath + "objectETag";
+  test_fixture f;
+  TH1F h1("objectETag", "objectETag", 100, 0, 99);
+  f.api.storeAsTFile(&h1, objectPath, f.metadata);
+
+  // then get the headers
   std::string etag;
   std::vector<std::string> headers;
   std::vector<std::string> pfns;
-  auto updated = CcdbApi::getCCDBEntryHeaders("http://ccdb-test.cern.ch:8080/TOF/LHCphase/1567080816927", etag, headers);
+  string path = objectPath + "/" + std::to_string(getCurrentTimestamp());
+  auto updated = CcdbApi::getCCDBEntryHeaders("http://ccdb-test.cern.ch:8080/" + path, etag, headers);
```

Comment:
ok. so I was wrong and we checked to supposedly existing object. This is now as it should be! Thanks.